### PR TITLE
Student edit

### PR DESCRIPTION
**Pull request by anguye2**
- Added Edit page where student can edit their name and email
- New email will be checked for validation (has '@' and '.')
- Once the student changed their name, studentDashboard.html will update their name after the submit button is clicked on
- Either field on edit page can be left blank
- Fixed some css buttons
To test: Add your username to students table. Sign in. Click on Edit button to edit your infomation

### CODE REVIEW

**Review by pdx-cs-lbryan — CHANGES_REQUESTED** [review]
please remove capstone360.db

**Review by pdx-cs-lbryan — CHANGES_REQUESTED** [review]
When trying to access student log in when not signed into CAS, I get an error message saying that no student is found. I should be taken to the CAS sign in page

File: student_dashboard.py [hunk 1]
```diff
@@ -23,3 +37,38 @@ def get(self):
...
+        return render_template('studentDashboard.html',
+                               name=student_name,
+                               user_name=user_name)
+# Edit Student class handles get requests from
+# student Dashboard when Edit is clicked on
+
+
+class EditStudent(MethodView):
+    @login_required
+    def get(self):
```
Comment (pdx-cs-lbryan) [inline student_dashboard.py:73]: method needs comments

File: gbmodel/model_sqlalchemy.py [hunk 2]
```diff
@@ -237,6 +237,19 @@ def check_team_lead(self, s_id):
         except exc.SQLAlchemyError:
             return False
 
+    # Allows students to edit their name and email address
+    # Input: student's new email and name and current user id
+    # Output: apply new name and email to students in student table
+    def edit_student(self, id, new_name, new_email):
+        stds = students.query.filter(students.id == id).all()
```
Comment (pdx-cs-lbryan) [inline gbmodel/model_sqlalchemy.py]: a try/except block would be useful here

**Comment by pdx-cs-lbryan** [comment]
Is your task complete?
-adds student editing info

Is the branch caught up to master/are there merge conflicts still to be resolved?
ahead of master

Are old files unnecessarily introduced?
No

Is old code unnecessarily overwritten?
No

Are new issues introduced to existing code? Do existing automated tests continue to work?

Log in: yes
Dashboard: yes
shows score range? looked to be working
Edit: yes
Review: yes
Prof View of Student Reports: yes
Report Generation session: yes
Report Generation individual: yes
Set/Add Date: yes
Automated Tests: pass

Uses 4 spaces where appropriate, not tabs?
yes

camelCase for class names, snake_case for variable and function names?
yes

code is commented?
some methods need comments

Does the code handle edge cases? Are try/except statements used sufficiently?
some methods need error handling

Is there documentation on how to test your code somewhere?
some

**Review by pdx-cs-lbryan — APPROVED** [review]
